examples/GAN/gan_mnist_pytorch/README.md

```diff
@@ -0,0 +1,16 @@
+
+*This folder contains the required files and the example code to demonstrate how to train a simple GAN on the MNIST dataset.*
+## The file version can be found on this [PyTorch Lightning GAN example](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/domain_templates/generative_adversarial_net.py)
+
+### Folders and Files
```

Comment:
Seems like we usually don't have any sub-folders here, so call these sections "Files" instead?